task_scheduler/configs/config.dev.sample.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
As you are adding files that are not python files, please be sure to add these path files on your MANIFEST.in file